_targets.R

```diff
@@ -0,0 +1,34 @@
+library(targets)
+source('code.R')
+tar_option_set(packages = c("tidyverse", "sbtools", "whisker"))
+list(
+  # Get the data from ScienceBase
+  tar_target(
+    model_RMSEs_csv,
+    fetch_data(project_output_dir = "1_fetch/out/model_RMSEs.csv"),
+    format = "file"
+  ), 
+  # Prepare the data for plotting
+  tar_target(
+    eval_data,
+    process_data(project_input_dir = model_RMSEs_csv),
```

Comment:
You've got this exactly right - passing `model_RMSEs_csv` to `process_data()`, but again you've got a bit of a mismatch between your argument name `project_input_dir` and what you're passing in (the raw data file). Can you edit the name of this argument to `process_data` to reflect what you're passing in?